bodhi/server/consumers/masher.py

```diff
@@ -902,6 +903,36 @@ def compose_atomic_trees(self):
         mashed_repos = dict([('-'.join(os.path.basename(repo).split('-')[:-1]), repo)
                              for repo in self.state['completed_repos']])
         for tag, mash_path in mashed_repos.items():
+            #  TODO: Find out the actual values for the tag
+            #  also for the code below, should we use the mash_path instead
+            #  of current_workdir?
+            #  * Pungi config repos hsa $arch, does it matter in bodhi?
+            #  * Do we need any other config files in the pungi-fedora config directory before we fireup the pungi
+            if tag.find('f26') != -1:  # This is a F26 build
+                current_workdir = os.path.join(self.path, self.id)
+                repo_dir = os.path.join(current_workdir, "pungi-fedora")
+                branch = "f26"
+                git_checkout(repo_dir, "https://pagure.io/pungi-fedora", branch)
```

Comment:
a fedora-infra-utils package with a python library for common retryables like this would be nice. is that something that exists or perhaps a feature request that can be made?